### PR TITLE
Add version number to docs URLs.

### PR DESCRIPTION
**Pull request by rcorral**

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @kennyt

File: src/js/constants/UnitSummaries.js [hunk 1]
```diff
@@ -1,89 +1,132 @@
...
-    documentation_url: `${Config.documentationURI}/overview/components/`
+    getDocumentationURI: function () {
+      return MetadataStore.buildDocsURI('/overview/components/');
+    }
   },
   'dcos-mesos-master.service': {
     summary: 'The Mesos master process orchestrates agent tasks.',
-    documentation_url: `${Config.documentationURI}/getting-started/installing/installing-enterprise-edition/troubleshooting/#scrollNav-2`
+    getDocumentationURI: function () {
+      return MetadataStore.buildDocsURI('/getting-started/installing/installing;-enterprise-edition/troubleshooting/#scrollNav-2');
```
Comment (MatApple) [inline src/js/constants/UnitSummaries.js]: Got a semicolon in the uri string

File: src/js/constants/UnitSummaries.js [hunk 2]
```diff
@@ -1,89 +1,132 @@
...
-    documentation_url: `${Config.documentationURI}/`
+    getDocumentationURI: function () {
+      return MetadataStore.buildDocsURI('/');
+    }
   },
   'dcos-gen-resolvconf.timer': {
     summary: 'Sets the dcos-gen-resolvconf.service to be run once a minute.',
-    documentation_url: `${Config.documentationURI}/getting-started/installing/installing-enterprise-edition/troubleshooting/#scrollNav-2`
+    getDocumentationURI: function () {
+      return MetadataStore.buildDocsURI('/getting-started/installing/installing;-enterprise-edition/troubleshooting/#scrollNav-2');
```
Comment (MatApple) [inline src/js/constants/UnitSummaries.js]: Another semi-colon

File: src/js/constants/UnitSummaries.js [hunk 3]
```diff
@@ -1,89 +1,132 @@
...
-    documentation_url: `${Config.documentationURI}/`
+    getDocumentationURI: function () {
+      return MetadataStore.buildDocsURI('/');
+    }
   },
   'dcos-gunicorn-bouncer.service': {
     summary: 'Processes login requests from users, as well as checking if an authorization token is valid.',
-    documentation_url: `${Config.documentationURI}/installing-enterprise-edition-1-6/security-and-authentication/`
+    getDocumentationURI: function () {
+      return MetadataStore.buildDocsURI('/installing-enterprise-edition-1-6/;security-and-authentication/');
```
Comment (MatApple) [inline src/js/constants/UnitSummaries.js]: semicolon - starting to wonder if these should be here?

File: src/js/constants/UnitSummaries.js [hunk 4]
```diff
@@ -1,89 +1,132 @@
...
-    documentation_url: `${Config.documentationURI}/`
+    getDocumentationURI: function () {
+      return MetadataStore.buildDocsURI('/');
+    }
   },
   'dcos-gen-resolvconf.service': {
     summary: 'Periodically writes /etc/resolv.conf so that only currently active Mesos masters with working Mesos DNS are in it.',
-    documentation_url: `${Config.documentationURI}/getting-started/installing/installing-enterprise-edition/troubleshooting/#scrollNav-6`
+    getDocumentationURI: function () {
+      return MetadataStore.buildDocsURI('/getting-started/installing/installing;-enterprise-edition/troubleshooting/#scrollNav-6');
```
Comment (MatApple) [inline src/js/constants/UnitSummaries.js]: semicolon

File: src/js/constants/UnitSummaries.js [hunk 5]
```diff
@@ -1,89 +1,132 @@
...
+    }
   },
   'dcos-mesos-slave-public': {
     summary: 'Runs a publicly accessible Mesos agent on the node.',
-    documentation_url: `${Config.documentationURI}/administration/dcosarchitecture/security/#scrollNav-3`
+    getDocumentationURI: function () {
+      return MetadataStore.buildDocsURI('/administration/dcosarchitecture/;security/#scrollNav-3');
+    }
```
Comment (MatApple) [inline src/js/constants/UnitSummaries.js]: ;

Comment (mlunoe) [inline src/js/constants/UnitSummaries.js]: 😉

File: src/js/stores/MetadataStore.js [hunk 6]
```diff
@@ -31,6 +32,13 @@ var MetadataStore = Store.createStore({
     this.removeListener(eventName, callback);
   },
 
+  buildDocsURI: function (path) {
+    let metadata = this.get('dcosMetadata');
+    let version = metadata && metadata.version || 'latest';
+    let docsVersion = version.replace(/(.*?)\.(.*?)\..*/, '$1.$2');
```
Comment (mlunoe) [inline src/js/stores/MetadataStore.js]: 👍 Nice

**Comment by mlunoe** [comment]
Looks good! After tests and the semi-colon incidence I say: 'Go!'
```js
FAIL  src/js/components/modals/__tests__/CliInstallModal-test.js (2.139s)
��� CliInstallModal ��� #getCliInstructions ��� it it returns different data depending on OS
  - TypeError: _MetadataStore2.default.buildDocsURI is not a function
        at getCliInstructions (src/js/components/modals/CliInstallModal.js:52:324)
        at Object.eval (src/js/components/modals/__tests__/CliInstallModal-test.js:72:37)
        at emitTwo (events.js:87:13)
        at process.emit (events.js:172:7)
        at handleMessage (internal/child_process.js:695:10)
        at Pipe.channel.onread (internal/child_process.js:440:11)
```

**Comment by kennyt** [comment]
tests failing

File: src/js/constants/UnitSummaries.js (hunk 1 above)
Comment (rcorral) [inline src/js/constants/UnitSummaries.js]: Fixed!

File: src/js/constants/UnitSummaries.js (hunk 2 above)
Comment (rcorral) [inline src/js/constants/UnitSummaries.js]: Fixed!

File: src/js/constants/UnitSummaries.js (hunk 3 above)
Comment (rcorral) [inline src/js/constants/UnitSummaries.js]: Fixed!

File: src/js/constants/UnitSummaries.js (hunk 4 above)
Comment (rcorral) [inline src/js/constants/UnitSummaries.js]: Fixed!

File: src/js/constants/UnitSummaries.js (hunk 5 above)
Comment (rcorral) [inline src/js/constants/UnitSummaries.js]: Fixed!

**Comment by MatApple** [comment]
LGTM

**Pull request merged**